fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -1662,6 +1683,16 @@ public Expression visitParenthesizedExpression(ParenthesizedExpressionContext ct
         return getExpression(ctx.expression());
     }
 
+    @Override
+    public Expression visitRowConstructor(RowConstructorContext ctx) {
+        return new Row(ctx.namedExpression().stream()
```

Comment:
since we do not support (a, b, c) to construct a struct, return a `OneRowRelation` in from `RowConstructorContext` is better

Reply:
yes，i fix it